docs/parcels.md

```diff
@@ -0,0 +1,149 @@
+# Parcels
+
+A single-spa parcel is a bundle of functionality (like an application or component) meant to be mounted manually 
+by an application. Parcels use similar methodology as applications but are mounted by a manual function call rather than the activity function.
+A parcel can be as large as an application or as small as an component and written in 
+any language as long as it exports the correct lifecycle events. In a single-spa world, your SPA contains 
+many registered applications and potentially many parcels. Typically we recommend you mount a parcel within 
+the context of an application because the parcel will be unmounted with the application.
+
+## Parcel configuration
+A parcel is just an object with 3 or 4 functions on it, it can be loaded syncronously or asyncronously.
+Each function is considered a lifecycle method, `bootstrap`, `mount`, `unmout`, `update` and must return a promise.
```

Comment:
`unmount` is spelled wrong.

Also, the "bootstrap, mount, unmount, and update part" of this sentence doesn't make sense in the context of the rest of the sentence.

> Each function on a parcel object is a lifecycle method, which is a function that returns a promise. Parcels have three required lifecycle methods (`bootstrap`, `mount`, and `unmount`) and one optional lifecycle method (`update`).